eleventy-fetch.js

```diff
@@ -34,11 +34,19 @@ function isFullUrl(url) {
 	}
 }
 
+function isAwaitable(maybeAwaitable) {
```

Comment:
I use this util in a couple of places. We could add a utils file that can be reused, instead, if we'd prefer.